css/style.css

```diff
@@ -0,0 +1,84 @@
+body {
+    background-color: whitesmoke;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+header {
+    padding: 15 25;
+    top: 0;
+    position: fixed;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: whitesmoke;
+    flex-direction: row;
+    width: 100%;
+}
+
+nav {
+    /* width: 30%; */
+}
+
+.logo > img {
+    max-width: 300;
+    margin-left: 20px;
+}
+
+nav > ul  {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+}
+
+nav > ul > li {
+    margin: 0 25;
+}
+
+#hero {
+    margin-top: 200px;
+    text-align: center;
+    
+}
+#hero > h2 {
+    margin-bottom: 20px;
+}
+
+#hero > form > #submit {
+    margin: 20px;
+}
+
+#feature {
+    text-align: left;
+    display: flex;
+    flex-direction: column;
+    justify-content: left;
+    align-items: left;
+}
+
+.grid {
```

Comment:
it would be better and more clean to use CSS grid to align child elements

Reply:
- [ ] [Required] Good shot! If you can add the reason for the requested changes here, it will make the comment perfect in my candid opinion. Please, say why this has to be done.

Reply:
it would be better and cleaner to use CSS grid to align child elements, and the reason for that is to lets us properly separate the order of elements in the source from their visual presentation.